Cargo.toml

```
@@ -16,14 +16,14 @@ clap = "4.5"
 fnv = "1.0"
 futures = { version = "0.3", features = ["compat"] }
 pin-project = "1.1"
-product-config = { git = "https://github.com/stackabletech/product-config.git", tag = "0.6.0" }
+product-config = { git = "https://github.com/stackabletech/product-config.git", tag = "0.7.0" }
 rstest = "0.19"
```

It's superficial since this sets the minimum version, but the vscode crate plugin shows it as red when it is still 0.19.

```suggestion
rstest = "0.21"
```

---

Ah, I was wondering if it would be alright to update other dependencies too :smile: done that now in https://github.com/stackabletech/hello-world-operator/pull/104/commits/2966812089d7d4798718026100e0d5aa7dafb515 thank you

---

Oh sorry, I didn't make that clear. Yeah, I mean it was probably ok since the code compiled. Other times the updates are necessary. But since we are bumping the operator version we might as well do all dependencies.